addons/chemical/XEH_postInit.sqf

```diff
@@ -0,0 +1,27 @@
+#include "script_component.hpp"
+
+//Events
+[QGVAR(gasCheck), {_this call FUNC(gasCheck)}] call CBA_fnc_addEventHandler;
```

Comment:
[QGVAR(gasCheck), LINKFUNC(gasCheck)] call CBA_fnc_addEventHandler;
should be changed into it (applies to other cba addEH